manifests/candlepin.pp

```diff
@@ -31,6 +28,8 @@
   }
 
   $java_client_cert_name = 'java-client'
+  $artemis_alias = 'artemis-client'
+  $artemis_client_dn = "C=${city}, ST=${state}, O=candlepin, OU=${org_unit}, CN=${cname}"
```

Comment:
I think the CN should be $hostname rather than $cname

Reply:
To be safe, I think this is correct. See https://github.com/theforeman/puppet-certs/pull/275/files#diff-0a129cc5a540f220caade2da4fc3229bR37

Reply:
Hmm, well here's the contents of the file:

```
katelloUser=C=Raleigh, ST=North Carolina, O=candlepin, OU=SomeOrgUnit, CN=[]
```

Hard to discern what you wanted me to see at that URL..

Reply:
Ohh, yea, I see now how this gets handled.

Reply:
Updated to hostname, I forget cname is *additional* cnames to add.

Reply:
In SSL you have `Common Name` (`cn`) and `subject Alternative Name` (`subjectAltName`). `cname` is DNS terminology.